src/components/AccountConfigDialog/ConnectAccountConfigDialog.jsx

```
@@ -0,0 +1,593 @@
+          <Text size="small" weight="regular" color="#414651">
+            {`Connecting your cloud account will allow ${serviceOrgName} to manage and
+            automate all associated instances.`}
+          </Text>
+        </ListItem>
+      </List>
+      <Box>
+        <Text
+          size="small"
+          weight="semibold"
+          color="#344054"
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)